arch/arm64/src/imx9/ddr/hardware/imx9_ddr_training.h

```diff
@@ -0,0 +1,152 @@
+/****************************************************************************
+ * arch/arm64/src/imx9/ddr/hardware/imx9_ddr_training.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_ARM64_SRC_IMX9_DDR_HARDWARE_IMX9_DDR_TRAINING_H
+#define __ARCH_ARM64_SRC_IMX9_DDR_HARDWARE_IMX9_DDR_TRAINING_H
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define MHZ(x) ((x) * 1000000UL)
```

Comment:
I don't think MHZ() is used anywhere, please remove it

Reply:
Good catch! fixed.